### ISSUE TITLE
Make use of image pairs

### ISSUE THREAD

**Issue by JoshVarty** [JoshVarty/KaggleClouds#13]
Let's try to figure out whether or not we can get any value out of the image pairs we've created. 

To start:

1. Train UNet on all train images. 
   - Test against only test images for which we have a pair.
   - Get LB score.
2. Train UNet on train images with a pair in the training set. Include label in input.
   - Test against only test images for which we have a pair.
   - Get LB score
3. Train UNet on train images with a pair in the test set. Include label AND image in input.
   - Test against only test images for which we have a pair.
   - Get LB score

**Comment by JoshVarty** [comment]
Baseline: **0.518**

Trained with labels: **0.520**

Trained with labels+paired image:

**Comment by JoshVarty** [comment]
Couldn't find a good way to use them :( :( :(
